analyzers/src/SonarAnalyzer.CSharp/LiveVariableAnalysis/CSharp/CSharpLiveVariableAnalysis.cs

```diff
@@ -178,6 +182,16 @@ private void ProcessIdentifier(IdentifierNameSyntax identifier, State state)
             }
         }
 
+        private void ProcessGenericName(SyntaxNode genericName, State state)
+        {
+            if (genericName.Parent.IsKind(SyntaxKind.InvocationExpression)
+                && semanticModel.GetSymbolInfo(genericName).Symbol is IMethodSymbol method
```

Comment:
Can be rewritten like: `&& semanticModel.GetSymbolInfo(genericName).Symbol is IMethodSymbol { MethodKind: MethodKindEx.LocalFunction } method`